### ISSUE TITLE
How to  test the dataset? What is the Command Line?

### ISSUE THREAD

**Issue by chenghan1995** [L1aoXingyu/reid_baseline#4]
I am not familiar with Python Fire. How to start the **test** function in main_reid.py? 
Thanks very much.

**Comment by chenghan1995** [comment]
Thanks very much. I‘ve solved it just now. Here is my command line
CUDA_VISIBLE_DEVICES=1 python3 main_reid.py test --save_dir='./pytorch-ckpt/market1501_softmax' --model_name='softmax' --load_model='./pytorch-ckpt/market1501_softmax/model_best.pth.tar'
